tests/py/test_search.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from gratipay.testing import Harness
+
+
+FULL = """\
+
+<p>Four score and seven years ago our fathers brought forth on this continent,
+a new nation, conceived in Liberty, and dedicated to the proposition that all
+men are created equal.</p>
+
+<p>Now we are engaged in a great civil war, testing whether that nation, or any
+nation so conceived and so dedicated, can long endure. We are met on a great
+battle-field of that war. We have come to dedicate a portion of that field, as
+a final resting place for those who here gave their lives that that nation
+might live. It is altogether fitting and proper that we should do this.</p>
+
+<p>But, in a larger sense, we can not dedicate&mdash;we can not
+consecrate&mdash;we can not hallow&mdash;this ground. The brave men, living and
+dead, who struggled here, have consecrated it, far above our poor power to add
+or detract. The world will little note, nor long remember what we say here, but
+it can never forget what they did here. It is for us the living, rather, to be
+dedicated here to the unfinished work which they who fought here have thus far
+so nobly advanced. It is rather for us to be here dedicated to the great task
+remaining before us&mdash;that from these honored dead we take increased
+devotion to that cause for which they gave the last full measure of
+devotion&mdash;that we here highly resolve that these dead shall not have died
+in vain&mdash;that this nation, under God, shall have a new birth of
+freedom&mdash;and that government of the people, by the people, for the people,
+shall not perish from the earth.</p>
+
+"""
+
+
+class Tests(Harness):
+
```

Comment:
Why? Again, this is a standard pattern in our test suite, and is basically just boilerplate. I would rather use py.test's dependency injected [fixtures](http://docs.pytest.org/en/latest/fixture.html) (we've had a good experience with them [in Aspen/Pando](https://github.com/AspenWeb/pando.py/tree/0.42/tests)), but we're too far along the class-based test path to change now. Better to minimize boilerplate if we can't avoid it entirely. :)